benchmarks/active_record/benchmark.rb

```diff
@@ -0,0 +1,33 @@
+require "harness"
+require "bundler/inline"
+
+gemfile do
+  source "https://rubygems.org"
+  gem "activerecord", "~> 6.0.3", ">= 6.0.3.6"
+  gem "sqlite3", "~> 1.4"
+end
+
+require "active_record"
+
+ActiveRecord::Base.establish_connection adapter: "sqlite3", database: ":memory:"
+
+ActiveRecord::Schema.define do
+  create_table :posts, force: true do |t|
+    t.string :title
+    t.string :body
+  end
+end
+
+class Post < ActiveRecord::Base; end
+post = nil
+50000.times { post = Post.create!(title: "hi", body: "hello!") }
+
+id = post.id
+# heat any caches
+Post.where(id: id).first.title
+
+run_benchmark(10) do
+  100.times do |i|
```

Comment:
I would suggest doing 1000 iterations here. A longer-running benchmark will reduce the noise.